### ISSUE TITLE
Swagger-ui 3.3.2 can not parse markdown's url

### ISSUE THREAD

**Issue by bungoume** [swagger-api/swagger-ui#3767]
Swagger-UI 3.3.2 seems to be unable to successfully parse markdown 's url.

| Q                               | A
| ------------------------------- | -------
| Bug or feature request?         |  Bug
| Which Swagger/OpenAPI version?  | 3.0.0
| Which Swagger-UI version?       | 3.3.2
| How did you install Swagger-UI? | cdnjs
| Which browser & version?        | chrome61
| Which operating system?         | macOS

using `https://cdnjs.cloudflare.com/ajax/libs/swagger-ui/3.3.2/swagger-ui-bundle.js`

### Demonstration API definition
`appspec.json`
```js
{
  "openapi":"3.0.0",
  "info":{
    "description":"- [repository URL](https://github.com/swagger-api/swagger-ui)",
    "version":"1.0.0"
  }
}
```

### Expected Behavior
`<a href="https://github.com/swagger-api/swagger-ui">repository URL</a>`
in swagger-ui 3.3.1.

### Current Behavior
`<a>repository URL</a>`
in swagger-ui 3.3.2.

**Comment by webron** [comment]
@owenconti Sanitization strikes again?

**Comment by shockey** [comment]
Fixed by #3768 🎉
